pkg/apis/application/v1alpha1/app_types.go

```diff
@@ -97,6 +101,14 @@ type AppSpecConfig struct {
 	Secret AppSpecConfigSecret `json:"secret,omitempty"`
 }
 
+// +k8s:openapi-gen=true
+type AppSpecInstall struct {
+	// +kubebuilder:validation:Optional
+	// +nullable
+	// SkipCRDs decides whether to install CRDs that might be supplied with the chart
+	SkipCRDs bool `json:"skipCrds,omitempty"`
```

Comment:
```suggestion
	SkipCRDs bool `json:"skipCRDs,omitempty"`
```

The JSON field should match the Go struct apart from the first char.